dateparser/languages/language.py

```diff
@@ -164,6 +168,43 @@ def _set_wordchars(self, settings=None):
     def _generate_dictionary(self, settings=None):
         self._dictionary = Dictionary(self.info, settings=settings)
 
+    def _generate_normalized_dictionary(self, settings=None):
+        self._normalized_dictionary = NormalizedDictionary(self.info, settings=settings)
+
+    def _get_simplifications(self, settings=None):
+        if not settings.NORMALIZE:
+            if self._simplifications is None:
+                self._generate_simplifications()
+            return self._simplifications
+        else:
+            if self._normalized_simplifications is None:
+                self._generate_normalized_simplifications()
+            return self._normalized_simplifications
+
+    def _generate_normalized_simplifications(self):
+        self._normalized_simplifications = []
+        for simplification in self.info.get('simplifications', []):
+            c_simplification = {}
+            key, value = list(simplification.items())[0]
+            key = normalize_unicode(key)
+            if isinstance(value, int):
+                c_simplification[key] = str(value)
+            else:
+                c_simplification[key] = normalize_unicode(value)
+            self._normalized_simplifications.append(c_simplification)
+
+    def _generate_simplifications(self):
+        self._simplifications = []
+        for simplification in self.info.get('simplifications', []):
```

Comment:
we can enclose lines 186-193 in a separate function and add some sort of flag to normalize values. And reuse that function in both places.